component/org.wso2.carbon.identity.api.otp.service.emailotp/src/gen/java/org/wso2/carbon/identity/api/otp/service/emailotp/dto/OTPValidationFailureReason.java

```diff
@@ -0,0 +1,142 @@
+/*
+ * Copyright (c) 2021, WSO2 Inc. (http://www.wso2.com).
+ *
+ * WSO2 Inc. licenses this file to you under the Apache License,
+ * Version 2.0 (the "License"); you may not use this file except
+ * in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ * http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+package org.wso2.carbon.identity.api.otp.service.emailotp.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+import com.fasterxml.jackson.annotation.JsonCreator;
+import io.swagger.annotations.ApiModel;
+import io.swagger.annotations.ApiModelProperty;
+import javax.validation.constraints.*;
+
+
+import io.swagger.annotations.*;
+import java.util.Objects;
+import javax.validation.Valid;
+import javax.xml.bind.annotation.*;
+
+public class OTPValidationFailureReason  {
+  
+    private String code;
+    private String message;
+    private String description;
+
+    /**
+    **/
```

Comment:
Shall we add a comment here?

Reply:
The files inside the `gen` directory are `auto-generated` files via `open-api` tool, I think we normally skip the styling in the `gen` directory files because they will get updated if we change the `YAML` file.